DB/src/main/java/io/deephaven/db/v2/parquet/ParquetTableWriter.java

```diff
@@ -442,10 +445,6 @@ public Integer get() {
                     }
                 }
             }
-            if (keyCount.intValue() == 0) {
```

Comment:
Just to check my understanding, if we had an empty dictionary we were writing a dummy one with a single empty string?

Reply:
Yeah.